LoopTests/PersistenceControllerTestCase.swift

```diff
@@ -8,38 +8,39 @@
 
 import XCTest
 import CoreData
-@testable import LoopKit
-
-class PersistenceControllerTestCase: XCTestCase {
-
-    var cacheStore: PersistenceController!
-
-    override func setUp() {
-        super.setUp()
-
-        cacheStore = PersistenceController(directoryURL: URL(fileURLWithPath: NSTemporaryDirectory(), isDirectory: true).appendingPathComponent(UUID().uuidString, isDirectory: true))
-    }
-
-    override func tearDown() {
-        cacheStore.tearDown()
-        cacheStore = nil
-
-        super.tearDown()
-    }
-
-    deinit {
-        cacheStore?.tearDown()
-    }
-
-}
-
-extension PersistenceController {
-    func tearDown() {
-        managedObjectContext.performAndWait {
-            let coordinator = self.managedObjectContext.persistentStoreCoordinator!
-            let store = coordinator.persistentStores.first!
-            let url = coordinator.url(for: store)
-            try! self.managedObjectContext.persistentStoreCoordinator!.destroyPersistentStore(at: url, ofType: NSSQLiteStoreType, options: nil)
-        }
-    }
-}
+// LoopKit is not compiled for Testing in carthage builds. LoopKit private units should be tested in LoopKit
```

Comment:
Please consider an alternate solution to commenting this out.

Reply:
See above.